docs/docusaurus/docs/oss/changelog.md

```diff
@@ -14,6 +14,23 @@ When we deprecate our public APIs, we will
 
 Before we completely remove the functionality in a new major release, there will be at least one minor release that contains the deprecation so that you can smoothly transition to the new API.
 
+### 1.2.3
+* [BUGFIX] double-sided z score renderer ([#10648](https://github.com/great-expectations/great_expectations/pull/10648))
+* [BUGFIX] Stop masking config strings ([#10625](https://github.com/great-expectations/great_expectations/pull/10625))
+* [DOCS] actions not ready for contributions ([#10646](https://github.com/great-expectations/great_expectations/pull/10646))
```

Comment:
We updated the status in the README because you currently can't contribute actions